### DISCUSSION TITLE
share database for public access

### DISCUSSION THREAD

**Discussion by I-r-a-j** [supabase/supabase#29309] [Questions] [answered]
Hi I am a beginner in supabase,

I want to know how to share a readable database tables and its quires to public with a url.

thanks

**Comment by GaryAustin1** [comment, marked as answer]
If you go to the API tab in the dashboard there is an option for BASH in upper right.  That will show you the REST http call needed for your tables with many examples on each table.

You would need RLS disabled or if enabled policies allowing anon role access.
